frontend/app/(home)/components/UseCases/components/UseCasesListing/UseCasesListing.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from "react";
+import { LuPanelLeft } from "react-icons/lu";
+
+import Spinner from "@/lib/components/ui/Spinner";
+import { useDevice } from "@/lib/hooks/useDevice";
+import { cn } from "@/lib/utils";
+
+import { UseCaseComponent } from "./components/UseCaseComponent";
+import { casesExamples } from "./data/cases";
+import { UseCase } from "./types";
+
+export const UseCasesListing = (): JSX.Element => {
+  const [cases, setCases] = useState<UseCase[]>([]);
+  const [selectedCaseId, setSelectedCaseId] = useState<string>();
+  const selectedCase = cases.find((c) => c.id === selectedCaseId);
+  const { isMobile } = useDevice();
+
+  useEffect(() => {
+    setCases(casesExamples);
+    setSelectedCaseId("1");
+  }, []);
+
+  if (selectedCaseId === undefined) {
+    return (
+      <div className="flex justify-center">
+        <Spinner />
+      </div>
+    );
+  }
+
+  return (
+    <div className="flex grid grid-cols-6 gap-10 flex flex-column items-start">
+      <div
+        className={cn(
+          "col-span-2 flex flex-col gap-3",
+          isMobile && "col-span-6"
```

Comment:
isMobile takes some time to detect the screen size and reload, I think we should rely as much as possible on tailwind's way to handle media queries `col-span-6 md:col-span-2`

Reply:
Agree 👍🏽